pyei/r_by_c.py

```diff
@@ -546,18 +651,23 @@ def _calculate_polarization(
             to calculate the associated threshold.
             """
             )
-        return threshold, percentile, samples, groups, candidate
+        return threshold
 
     def polarization_report(
-        self, groups, candidate, threshold=None, percentile=None, verbose=True
-    ):
+        self,
+        groups: list[int],
+        candidate: int,
```

Comment:
```suggestion
        candidate: str,
```

Right? That is what the docstring says